chroma-manager/tests/feature/cli/features/steps/standard_cli.py

```diff
@@ -3,6 +3,7 @@
 # Copyright (c) 2012 Whamcloud, Inc.  All rights reserved.
 # ========================================================
 
+from __future__ import print_function
```

Comment:
have removed the header (test code where similar files have no header) and removed the changes